backend/database/migrations/versions/0a5eb83e9af2_convert_content_metadata_from_json_to_.py

```
@@ -0,0 +1,55 @@
+               existing_nullable=True)
+    op.drop_index(op.f('ix_scenario_personas_deleted_at'), table_name='scenario_personas')
+    op.create_table('langchain_pg_collection',
+    sa.Column('name', sa.VARCHAR(), autoincrement=False, nullable=True),
+    sa.Column('cmetadata', postgresql.JSON(astext_type=sa.Text()), autoincrement=False, nullable=True),
+    sa.Column('uuid', sa.UUID(), autoincrement=False, nullable=False),
+    sa.PrimaryKeyConstraint('uuid', name='langchain_pg_collection_pkey'),
+    postgresql_ignore_search_path=False
+    )
+    op.create_table('langchain_pg_embedding',
+    sa.Column('collection_id', sa.UUID(), autoincrement=False, nullable=True),
+    sa.Column('embedding', pgvector.sqlalchemy.Vector(), autoincrement=False, nullable=True),
```

Ruff (0.14.1)

Undefined name `pgvector`

(F821)